net/uwsgi-cgi/Makefile

```diff
@@ -26,6 +26,15 @@ define Package/uwsgi-cgi
   DEPENDS:=+libopenssl +libpcre +jansson +libuuid +libxml2
 endef
 
+define Package/uwsgi-cgi-luci-support
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=Web Servers/Proxies
+  TITLE:=Support files for LuCI on Nginx
+  URL:=http://unbit.com/
+  DEPENDS:=uwsgi-cgi
```

Comment:
Should it be with + so that you can just select uwsgi-cgi-luci-support and also uwsgi-cgi gets selected? now you have first select uwsgi-cgi and only then you can even see uwsgi-cgi-luci-support in menuconfig